config/optional/field.field.user.user.field_password_expiration.yml

```diff
@@ -0,0 +1,26 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.user.field_password_expiration
+  enforced:
+    module:
+      - password_policy
+  module:
+    - user
+id: user.user.field_password_expiration
+field_name: field_password_expiration
+entity_type: user
+bundle: user
+label: 'Password Expiration'
+description: 'The password will expire based on the number of days defined in the policy'
+required: false
+translatable: false
+default_value:
+  -
+    value: 1 
```

Comment:
This option doesn't work when you create new user.

For example:
1. Create new user
1.a. check `Password Expiration` option
1.b. you can also set date in past
2. Save user
3. Edit user
3.a. Option will be unchecked
3.b. Date will be current date

And label is also a bit misleading, because when you edit user and check that it will be automatically marked as expired and it's not related to expire date.

Reply:
For now, we remove the config and ship with the default where this is not checked. I will try to push the drupal.org issue regarding the misleading wording. See https://www.drupal.org/project/password_policy/issues/2990528